dev/Dockerfile.dev

```diff
@@ -22,8 +22,6 @@ COPY setup.cfg .
 COPY dev/entrypoint.sh ./
 
 RUN chmod 777 -R .
-RUN chmod 777 /root
-RUN chmod 777 -R /usr/local/lib/
```

Comment:
I'm sure there was a reason for this. But we can remove it for now and see when problems pop up.

Reply:
@peb-adr We removed the 777-right settings from backends Dockerfile.**dev**